From 53d91a827c397880e4b727104efba64eda87d790 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Fri, 5 Jan 2018 13:41:44 +0200
Subject: [PATCH 3178/5242] MLK-17280-1: drm: panel: rm67191: Fix power on
 sequence

commit  e786da7354b8650953924fd1092d6e23d9dfbf2b from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the vendor driver sample there is a sleep after the exit
sleep and display on commands, but it seems that these sleeps are only
causing stability issues when the display signal is sent to the panel,
so remove them.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/panel/panel-raydium-rm67191.c |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/panel/panel-raydium-rm67191.c b/drivers/gpu/drm/panel/panel-raydium-rm67191.c
index 4e9fea9..4f41fd3 100644
--- a/drivers/gpu/drm/panel/panel-raydium-rm67191.c
+++ b/drivers/gpu/drm/panel/panel-raydium-rm67191.c
@@ -296,16 +296,12 @@ static int rad_panel_prepare(struct drm_panel *panel)
 		goto fail;
 	}
 
-	msleep(120);
-
 	ret = mipi_dsi_dcs_set_display_on(dsi);
 	if (ret < 0) {
 		DRM_DEV_ERROR(dev, "Failed to set display ON (%d)\n", ret);
 		goto fail;
 	}
 
-	msleep(100);
-
 	rad->prepared = true;
 
 	return 0;
-- 
1.7.9.5

